src/components/ChangePlan/Checkout/BillingInformation/BillingInformation.test.js

```diff
@@ -0,0 +1,432 @@
+import React from 'react';
```

Comment:
Perhaps we can go through the tests with these recommendations. They become easier to test and read as well. https://github.com/FromDoppler/doppler-webapp/pull/1504

Reply:
Good, I will review it and add/update tests.